package.json

```diff
@@ -22,15 +29,19 @@
   "author": "Liran Tal <liran.tal@gmail.com> (http://www.enginx.com)",
   "license": "MIT",
   "dependencies": {
-    "lodash": "^4.16.2",
     "request": "^2.75.0"
   },
   "devDependencies": {
+    "babel-core": "^6.22.1",
+    "babel-loader": "^6.2.10",
+    "babel-preset-es2015": "^6.22.0",
     "coveralls": "^2.11.6",
     "istanbul": "^0.4.4",
     "mocha": "^3.1.0",
     "mocha-lcov-reporter": "^1.2.0",
     "nock": "^8.0.0",
-    "should": "^11.1.0"
+    "rimraf": "^2.6.1",
+    "should": "^11.1.0",
+    "webpack": "^1.14.0"
```

Comment:
Let's finish this milestone of webpack1, and then I promise the next PR will be the refactoring to webpack2. Since I didn't used webpack2 yet